### PR TITLE
chore: Add observability

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

File: ipc/bootstrap.sh [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+  script_name="$(basename $0)"
+  cat <<HELP
+Usage: ${script_name} --network <ENV> --name <NAME> --ip <IP> --halt-height <HEIGHT>
 Bootstrap IPC validator node.
 
-  --env          Environment where to run validator. Only kras and dar are allowed for now
-  --name         Human readable validator name.
-  --ip           IP address where IPC p2p is accessible from the internet
-  --halt-height  Height at which stop to produce blocks
+  --env | --network    Environment where to run validator. Only kras and dar are allowed for now
```
Comment (folex) [inline ipc/bootstrap.sh:14]:
```suggestion
  --env | --network    Environment where to run validator. Possible values: kras for mainnet, dar for testnet
```

File: ipc/bootstrap.sh [hunk 2]
```diff
@@ -78,10 +89,11 @@ while (($#)); do
   case "$1" in
   --env)
```
Comment (folex) [inline ipc/bootstrap.sh:90]: Was there an intention to allow for both `--env` and `--network`? If so, I guess gotta do this:

```suggestion
  --env)
  ;;
  --network)
```

Comment (nahsi) [reply ipc/bootstrap.sh]: > Was there an intention to allow for both `--env` and `--network`? 

Yep, for compatibility with docs and everything. `network` fits better as this is what we have in fcli, right?

File: ipc/README.md [hunk 3]
```diff
@@ -22,6 +22,13 @@ See `./boostrap.sh -h` for more info.
 docker compose up -d
 ```
 
+## Run with observabiliy stack
+
+```shell
+./boostrap.sh --network NETWORK --name NAME --ip IP --basicauth_username USERNAME --basicauth_password PASSWORD
```
Comment (folex) [inline ipc/README.md:28]: Where should I get the USERNAME and PASSWORD?

**Review by folex — APPROVED** [review]

**Pull request merged**